cc3d/core/PySteppables.py

```diff
@@ -1776,11 +2155,15 @@ def invariantDistanceVectorBetweenCells(self, _cell_from, _cell_to):
 
     def invariant_distance_vector_between_cells(self, cell1, cell2):
         """
-        This function will calculate distance vector between  COM's of cells . Respects boundary conditions
-        :return: {ndarray} distance vector
+        This function will calculate distance vector between  COM's of cells. Respects boundary conditions
```

Comment:
This fcn says it respects BC, but `invariant_distance_vector` says it assumes periodic BC. I believe `invariant_distance_vector` does respect BC as it calls `CompuCell.BoundaryStrategy.getInstance()`

Reply:
Addressed in https://github.com/CompuCell3D/CompuCell3D/pull/457/commits/238537aec7b36485a8268f7884ea9abdc0d80f0d